.travis.yml

```diff
@@ -1,8 +1,34 @@
+conditions: v1
 language: python
-sudo: required
-dist: xenial
-python: "3.7"
-cache: pip
+python:
+  - "3.6"
 before_install:
 - pip install -r requirements.txt
 script: pre-commit run --all-files
+
+jobs:
+  include:
+  - python: 3.7
+    dist: xenial
```

Comment:
It's probably okay to use xenial for everything

Reply:
Sorry, don't understand where I use `xenial`, because I think I use it only in this place.

Reply:
Yes but I'm telling you that you should just use it globally instead.